tests/generator/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,53 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`anonymous.js 1`] = `
+====================================options=====================================
+parsers: ["babel", "typescript", "flow"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+const f1 = function* () {
+  yield 0;
+};
+  
+const f2 = function * () {
+  yield 0;
+};
+
+const f3 = function* () {
+};
+  
+(function* () {
+  yield 0;
+});
+  
+(function * () {
+  yield 0;
+});
+
+(function* () {
+});
+  
+=====================================output=====================================
+const f1 = function* () {
+  yield 0;
+};
+
+const f2 = function* () {
+  yield 0;
+};
+
+const f3 = function* () {};
```

Comment:
So do we not always add a space after `function`?

Reply:
Generator&nbsp;functions are&nbsp;covered by&nbsp;#7028.

Reply:
The title of this PR and the changelog both say a space is **always** added after `function`, so it probably should really be added always. Otherwise, if we proceed with #7028 and if it isn't in the same release, users are going to face two waves of reformatting of anonymous generators: first `function*() → function* ()` and then `function* () → function *()`. As nobody wants this to happen, I take it we shouldn't proceed with this PR until we have consensus on what to do with #7028.